homeassistant/components/abode/alarm_control_panel.py

```diff
@@ -27,8 +27,7 @@
     data = hass.data[DOMAIN]
 
     async_add_entities(
-        [AbodeAlarm(data, await hass.async_add_executor_job(data.abode.get_alarm))],
-        True,
+        [AbodeAlarm(data, await hass.async_add_executor_job(data.abode.get_alarm))]
```

Comment:
Removed `True` to resolve the issue with the alarm control panel not showing up in the integrations UI after adding the abode integration. I'm not sure what's causing the issue, but I'm also not sure why HA needs to refresh the device when it's added (as I understand that's why the `True` argument does). The reason being is this is occurring at start up or when the Abode integration is added, which the device states would be updated just seconds ago.